client/my-sites/plans-grid/components/actions.tsx

```diff
@@ -119,9 +118,23 @@ const SignupFlowPlanFeatureActionButton = ( {
 	}
 
 	return (
-		<Button className={ classes } onClick={ handleUpgradeButtonClick } busy={ busy }>
-			{ btnText }
-		</Button>
+		<div className="plan-features-2023-grid__actions-button-container">
+			<Button
+				className={ classes }
+				onClick={ () => handleUpgradeButtonClick( false ) }
```

Comment:
Similar here. Let's wrap this with `useCallback` to force a single reference across rerenders. and also for the case below

Of course, let me know if you think otherwise. :-)

Reply:
Eh, I do have my reservations with useCallback ([see this post](https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse)).

The gains are marginal and we're introducing the overhead of calling `useCallback` twice as we have two different click responses there so it's impossible _not_ to define a new arrow function unless we use callback twice... Which really kills the point I think?